Airports/EGPE/Sector.txt

```diff
@@ -8,19 +8,29 @@ DISPLAY:Moray:Moray:Only PEAPP
 DISPLAY:Hebrides:Hebrides:Only PEAPP
 DISPLAY:Montrose:Montrose:Only PEAPP
 
+CIRCLE_SECTORLINE:PEAPP:EGPE:10
+
```

Comment:
Could you just move this one underneath the CIRCLE_SECTORLINE:PETWR please? Won't change functionality, just think it makes more sense TWR then APP 😄 #picky